### PR TITLE
Ci fix

### LINKED ISSUE #1249: CI Fix
## Description
Fix broken CI:
1) Update the docker image library to fms 2022.01
2) Add a coupled model test to rt_gnu.conf that excludes the wave component. This is necessary because, with waves, there are not enough cores available on aws using a single node.

### PR DESCRIPTION
**Pull request by MinsukJi-NOAA**
# PR Checklist

- [x] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [x] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsibility to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Fixes broken CI: 1) Update fms library in docker image; 2) Add cpld noaero nowave test back to rt_gnu.conf. This is to reduce the number of cores used by CI on 1-node AWS EC2 instance.

New baselines are needed for the 3 new tests added to rt.conf:
cpld_control_noaero_p8
cpld_control_nowave_noaero_p8
cpld_debug_noaero_p8

New baselines are needed for the 1 new test added to rt_gnu.conf:
cpld_control_nowave_noaero_p8

### Issue(s) addressed

- fixes #1249

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

None

### CODE REVIEW

File: tests/tests/cpld_control_c96_noaero_p8 [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+export INPES=$INPES_cpl_dflt
+export JNPES=$JNPES_cpl_dflt
+export THRD=$THRD_cpl_dflt
+export WRTTASK_PER_GROUP=$WPG_cpl_dflt
+
+OCN_tasks=$OCN_tasks_cpl_dflt
+ICE_tasks=$ICE_tasks_cpl_dflt
+export WAV_tasks=0
+export TASKS=$(( INPES*JNPES*6 + 1*WRTTASK_PER_GROUP + OCN_tasks + ICE_tasks ))
```
Comment (BrianCurtis-NOAA) [inline tests/tests/cpld_control_c96_noaero_p8]: the `1*` seems redundant

**Review by BrianCurtis-NOAA — APPROVED** [review]

File: tests/tests/cpld_control_c96_noaero_p8 (hunk 1 above)
Comment (MinsukJi-NOAA) [reply tests/tests/cpld_control_c96_noaero_p8]: It represents `WRITE_GROUP`, but I will check again if `WRITE_GROUP` is defined here.

Comment (BrianCurtis-NOAA) [reply tests/tests/cpld_control_c96_noaero_p8]: then you could replace `6` with NTILES if its defined as well.

File: tests/rt_gnu.conf [hunk 2]
```diff
@@ -66,6 +66,9 @@ RUN     | cpld_control_noaero_p8
 COMPILE | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                              |                | fv3         |
 RUN     | cpld_debug_noaero_p8                                                                                                    |                | fv3         |
 
+COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                                         |                | fv3         |
```
Comment (DeniseWorthen) [inline tests/rt_gnu.conf]: Why do we need a new compile line here?

Comment (MinsukJi-NOAA) [reply tests/rt_gnu.conf]: I meant to use `-DAPP=S2S` in this line. But, alternatively, I can use the compile line 63 instead.

File: tests/tests/cpld_control_c96_noaero_p8 [hunk 3]
```diff
@@ -0,0 +1,105 @@
+#
+#  cpld_control C96 noaero P8 test
```
Comment (DeniseWorthen) [inline tests/tests/cpld_control_c96_noaero_p8]: Should we call this "nowave_noareo_p8", or perhaps "s2s_p8" (where the std config is s2swa).

Comment (MinsukJi-NOAA) [reply tests/tests/cpld_control_c96_noaero_p8]: I think `nowave_noaero_p8` is consistent with existing tests such as `cpld_control_noaero_p8`. We could change to `s2s_p8` in the future when we update all test names to be consistent.

File: tests/tests/cpld_control_c96_noaero_p8 (hunk 1 above)
Comment (MinsukJi-NOAA) [reply tests/tests/cpld_control_c96_noaero_p8]: Done

**Comment by SamuelTrahanNOAA** [comment]
This has been merged into https://github.com/ufs-community/ufs-weather-model/pull/1257

**Comment by junwang-noaa** [comment]
Merged through PR#1257.

**Pull request closed without merging**